UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs

```diff
@@ -6,6 +6,7 @@
 using Objects;
 using Objects.Disposals;
 using Random = UnityEngine.Random;
+using Assets.Scripts.Messages.Server.SoundMessages;
```

Comment:
all scripts are under ``Assets/Scripts`` so you can get rid of that part when doing namespaces.

Reply:
That's the namespace that AudioSourceParameters is using, weirdly.  Should I change that?

Reply:
yes. The person who did it probably uses VS and VS auto suggest the whole path.